doc/rst/usingchapel/chplenv.rst

```diff
@@ -716,6 +716,18 @@ CHPL_LIB_PIC
 
    If unset, ``CHPL_LIB_PIC`` defaults to ``none``
 
+Character Set
+-------------
+   Chapel is tested with the Unicode character set and the traditional
```

Comment:
One might interpret "Chapel is tested" as recommending how to test Chapel rather than making a statement about what configuration is tested nightly and "supported". Can you think of a simple way to fix it? For example it could say something like "We have the most experience running Chapel with these character set settings." Or something like that. I'd also consider adding a `.. note ::` that points out that this may change in the future.

Reply:
I like that suggestion.  I'll use it.